src/Nethermind/Nethermind.Blockchain.Test.Runner/Program.cs

```diff
@@ -79,7 +79,7 @@ private static void WrapAndRunDirectoryStateTests(IStateTestRunner stateTest)
             var result = stateTest.RunTests().ToList();
             var failedTestsInCategory = result.Where(r => !r.Pass).Select(t => t.Name + " " + t.LoadFailure).ToArray();
             AllFailingTests.AddRange(failedTestsInCategory);
-            long categoryTimeInMs = result.Sum(t => t.TimeInMs);
+            long categoryTimeInMs = (long)result.Sum(t => t.TimeInMs);
```

Comment:
should those also be double's now?

Reply:
Doesn't matter too much since is the sum of all the doubles; so the decimals aren't going to be too relevant